src/render/dom/utils/build-html-styles.ts

```diff
@@ -133,25 +161,10 @@ export function buildHTMLStyles(
                 enableHardwareAcceleration,
                 allowTransformNone
             )
-        } else {
-            style.transform = createDeltaTransform(deltaFinal!, treeScale!)
-            if (transformTemplate)
-                style.transform = transformTemplate(transform, style.transform)
```

Comment:
I don't understand where we apply the transformTemplate now?

Reply:
Ah yes I believe the answer is we don't - this is fixed now